app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -119,4 +139,20 @@ def select_host(vm)
   def password_auth(provider)
     provider.authentication_userid_passwords.first
   end
+
+  def get_first_vm_network_id(conn, dc_id)
```

Comment:
I thought rubocop use to complain about using get/set method names.  In the past we tried to avoid them.  Ref https://github.com/bbatsov/ruby-style-guide#accessor_mutator_method_names.

These two `get_` methods look straightforward to change.  The `set_` here I could live with.

Glad to see these methods already in the private section of the class.  👍